### PR TITLE
feat(Notification): add new `bq-notification` component

### PR DESCRIPTION
**Pull request by dgonzalezr**
CleanShot 2023-05-02 at 18 20 42

### CODE REVIEW

**Review by endv-bogdanb — DISMISSED** [review]

**Comment by dgonzalezr** [comment]
# ☁️ Nx Cloud Report

We didn't find any information for the current pull request with the commit a3b26857a503edaf1138f119603561509fa66859.
Please verify you are running the latest version of the [NxCloud runner](https://www.npmjs.com/package/@nrwl/nx-cloud).

Check [the Nx Cloud Github Integration documentation](https://nx.dev/nx-cloud/set-up/github) for more information.

Alternatively, you can contact us at [cloud-support@nrwl.io](mailto:cloud-support@nrwl.io).

---

Sent with 💌 from [NxCloud.](https://nx.app)

**Review by endv-bogdanb — APPROVED** [review]

File: packages/bee-q/src/components/notification/bq-notification.tsx [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+@Component({
+  tag: 'bq-notification',
+  styleUrl: './scss/bq-notification.scss',
+  shadow: true,
+})
+export class BqNotification {
+  // Own Properties
+  // ====================
+
+  // private autoDismissTimeout: number;
```
Comment (endv-bogdanb) [inline packages/bee-q/src/components/notification/bq-notification.tsx:28]:
```suggestion
```

File: packages/bee-q/src/components/notification/bq-notification.tsx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  onNotificationHide() {
+    try {
+      notificationPortal.removeChild(this.el);
+      // Remove the notification portal from the DOM when there are no more notifications
+      if (notificationPortal.querySelector('bq-notification') === null) {
+        notificationPortal.remove();
+      }
+    } catch (error) {
+      if (error instanceof DOMException) return;
+      console.error(error);
```
Comment (endv-bogdanb) [inline packages/bee-q/src/components/notification/bq-notification.tsx:124]: We can also `throw error` here and let the default behaviour. Not sure if is better though

**Review by endv-bogdanb — DISMISSED** [review]

File: packages/bee-q/src/components/notification/bq-notification.tsx (hunk 1 above)
Comment (dgonzalezr) [reply packages/bee-q/src/components/notification/bq-notification.tsx]: Removed

**Review by endv-bogdanb — APPROVED** [review]

**Review by endv-bogdanb — APPROVED** [review]

**Pull request merged**